src/Expensify.js

```diff
@@ -116,7 +115,9 @@ class Expensify extends PureComponent {
             && !_.isEmpty(this.props.betas)
             && lodashGet(this.props, 'session.redirectToWorkspaceNewAfterSignIn', false)) {
             setRedirectToWorkspaceNewAfterSignIn(false);
-            Navigation.navigate(ROUTES.WORKSPACE_NEW);
+
+            // Name and create the workspace
```

Comment:
NAB but not a super useful comment, should explain why we are doing something not what we are doing.